### PR TITLE
Remove redundant and incorrect riot-desktop install instructions

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,9 +188,9 @@ Running as a Desktop app
 
 Riot can also be run as a desktop app, wrapped in electron. You can download a
 pre-built version from https://riot.im/desktop.html or, if you prefer,
```
Comment (jryans) [inline README.md]: Maybe also update URL to: https://riot.im/download/desktop/

File: README.md [hunk 2]
```diff
@@ -31,14 +31,8 @@ Note that Chrome does not allow microphone or webcam access for sites served
...
 
-### Desktop Installation for Debian Stretch
-
-1. Add the repository to your sources.list using either of the following two options:
-  - Directly to sources.list: `echo "deb https://riot.im/packages/debian/ stretch main" | sudo tee -a /etc/apt/sources.list`
-  - As a separate entry in sources.list.d: `echo "deb https://riot.im/packages/debian/ stretch main" | sudo tee /etc/apt/sources.list.d/riot.list`
-2. Add the gpg signing key for the riot repository: `curl -s https://riot.im/packages/debian/repo-key.asc | sudo apt-key add -`
-3. Update your package lists: `sudo apt-get update`
-4. Install Riot: `sudo apt-get install riot-web`
+To install Riot as a destktop application, see [Running as a desktop
```
Comment (jryans) [inline README.md]:
```suggestion
To install Riot as a desktop application, see [Running as a desktop
```

**Comment by jryans** [comment]
I changed the base to `develop` but the preview doesn't seem to update... Please make sure it still makes sense when applied to `develop` before merging.

**Comment by richvdh** [comment]
> I changed the base to develop but the preview doesn't seem to update, maybe because of conflicts. Please make sure it still makes sense when applied to develop before merging.

I thought it might make sense to commit this to master, otherwise the instructions are going to be wrong until the next release.

**Comment by jryans** [comment]
> I thought it might make sense to commit this to master, otherwise the instructions are going to be wrong until the next release.

Ah, I see. We recently changed the default branch of the `riot-web` repo to `develop`, so going to https://github.com/vector-im/riot-web will show `develop`'s README by default now. I think `master` is less likely to be viewed via casual browsing these days at least, so I think targeting `develop` is best to reach most people.

**Comment by richvdh** [comment]
> Ah, I see. We recently changed the default branch of the riot-web repo to develop, so going to https://github.com/vector-im/riot-web will show develop's README by default now

Ah I see. Will fix the conflicts against develop then.

(what's the point of `master` if the default branch is `develop`?)

**Comment by t3chguy** [comment]
master is always the latest stable release branch

**Comment by jryans** [comment]
> (what's the point of `master` if the default branch is `develop`?)

The `master` branch matches the last released / stable version. `develop` contains the latest changes as they land.

**Pull request merged**